content/blog/2016/2016-11-16-blueocean-beta-declarative-pipeline-pipeline-editor.adoc

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: "Announcing the Blue Ocean beta, Declarative Pipeline and Pipeline Editor"
+tags:
+- blueocean
+- ux
+- pipeline
+- jenkinsworld
+author: i386
+---
+
+At Jenkins World on Wednesday 14th of September the Jenkins project was happy to
+introduce the beta release of Blue Ocean. Blue Ocean is the new user experience
+for Jenkins is built from the ground up to take advantage of Jenkins Pipeline
```

Comment:
"is the new user experience for Jenkins is built"? =) I assume there's a missing end-of-one-sentence-start-of-another.